clang/test/CXX/drs/dr18xx.cpp

```diff
@@ -287,6 +287,11 @@ namespace dr1837 { // dr1837: 3.3
       };
     };
   };
+  /* since-cxx11-warning@-6{{address of function '[] {
```

Comment:
Can you convert this to use a marker? We don't want readers to count relative offsets. In other words, it should read `since-cxx11-warning@#dr1837-lambda {{address of function ...`

You can see an example in the test for 1890.